packages/libs/react-components/src/components/Input/InputGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import { AlertBox } from '../Icons/svgs/AlertBox';
+import { AlertCircleOutline } from '../Icons/svgs/AlertCircleOutline';
+
+import {
+  StyledHelper,
+  StyledInfo,
+  StyledInputGroupHeader,
+  StyledInputGroupWrapper,
+  StyledInputs,
+  StyledLabel,
+  StyledTag,
+} from './styles';
+
+import classnames from 'classnames';
+import React, { FC } from 'react';
+
+export interface IInputGroupProps {
+  label?: string;
+  tag?: string;
+  info?: string;
+  helper?: string;
+  status?: 'success' | 'error';
+  disabled?: boolean;
+  children: React.ReactNode;
+  className: string;
```

Comment:
what we need this classname for?
is this for extending / overwriting classes and variants?

I think this is something we dont want, as we want the styles to be strict

Reply:
yes exactly, it's for extending, but i don't think it's a good idea to set a strict rule like this, cause in the end some styles for different usages can be different, or at least there should be room for modification of styles if we don't allow styled component extension, the consumer will wrap it in another element to overwrite it, which messes up the accessibility so I would say having extension ability would be beneficial

Reply:
hm..yeah i understand. 
I do have the opinion that this package, at the moment is for Kadena UI alone, so keep it as strict until we really need it.
If we open up the classname a developer can do what they want, where maybe the style system does not allow it (lets say, make the text green)
If the UI team dont give the classname option, and the developer wants something that is not possible, they need to go talk to the UI team. if this can be implemented. where, in the case of the green text, they probably say no.

I think a design system should not be overwritten.
But we can discuss with Eileen.

Reply:
remove classnames